src/components/TextField/TextField.jsx

```diff
@@ -1,15 +1,46 @@
 /* eslint-disable react/prop-types */
-import React from 'react';
+import React, { Fragment } from 'react';
 import { Input } from './style';
 
 const Textfield = (props) => {
-  const {
-    placeholder, disabled, error, bgcolor,
-  } = props;
+  const { onChange, value, error } = props;
   return (
+    <>
+      <Input type="text" value={value} onChange={onChange} />
+      <p>{error}</p>
+    </>
 
-    <Input type="text" placeholder={placeholder} disabled={disabled} error={error} bgcolor={bgcolor} />
 
   );
 };
-export default Textfield;
+const SelectField = (props) => {
```

Comment:
![image](https://user-images.githubusercontent.com/38630387/76597589-d2526800-6526-11ea-9b5e-dfd141387d79.png)
It should be like this.